src/core/Akka.Remote/Remoting.cs

```diff
@@ -114,18 +114,18 @@ internal interface IPriorityMessage { }
     internal sealed class Remoting : RemoteTransport
     {
         private readonly ILoggingAdapter _log;
-        private volatile IDictionary<string, HashSet<ProtocolTransportAddressPair>> _transportMapping;
-        private volatile IActorRef _endpointManager;
+        private IDictionary<string, HashSet<ProtocolTransportAddressPair>> _transportMapping;
+        private IActorRef _endpointManager;
 
         // This is effectively a write-once variable similar to a lazy val. The reason for not using a lazy val is exception
         // handling.
-        private volatile HashSet<Address> _addresses;
```

Comment:
Why replace this with a `Volatile.Write` down below?

Reply:
Its a multi read and only single write field.
volatile as perf issues if something is accessing it multiple times